core/src/test/java/org/testcontainers/containers/ExposedHostTest.java

```diff
@@ -29,7 +31,10 @@ public static void setUpClass() throws Exception {
         });
 
         server.start();
-        Testcontainers.exposeHostPorts(server.getAddress().getPort());
+        
+        Map<Integer, Integer> portMapping = new HashMap<>();
+        portMapping.put(server.getAddress().getPort(), 80);
```

Comment:
Please do not change the old test, it is still valid

Reply:
I’m not sure what you mean.

You requested a delegate for exposeHostPort(int port) and I implemented it.
So testing the not delegated method should be preferred?

Reply:
`Testcontainers.exposeHostPorts(server.getAddress().getPort());` is a valid use and should be tested.

Please add a test case, instead of changing the old one

Reply:
done

Reply:
Did you mean a new test class, or just another test method, @bsideup?

Reply:
just another method, actually 😅

Reply:
ok, next try